docs/relying-parties/reference/sub-plat-contentful.md

```diff
@@ -23,201 +16,85 @@ You can log into Contentful through [Mozilla Dashboard](https://sso.mozilla.com/
+| Capability       | **Required.** Capabilities enabled by the subscription product that services honor, and that customers are entitled to depending on the offering purchased.<br /> - Product managers are to provide the name(s) of capabilities and the Stripe Product ID (from Stripe) to SubPlat Engineers. <br /> - SubPlat Engineers are to add capabilities (and link the related service) into Contentful. <br /> - Product managers are to add capability(ies) to the Capabilities field in Offering. |
+| Common content   | Optional. Content used on multiple pages (e.g., Privacy Notice URL, TOS URL, Success Action Button URL, etc.), not specific to a certain component. If the product offering is available in other locales, translation fields appear in this entry, once locales have been selected.<br/>- Product managers are to create and add this entry to Offering, if applicable.                                                                                                                     |
+| Coupon Config    | Optional. Coupon configuration options of the Stripe Promotion Codes valid in specified countries for the product offering. <br/>- Product managers are to create and add this entry to Offering, if applicable.                                                                                                                                                                                                                                                                             |
+| IAP              | Optional. IAP configurations that include the Apple App Store productIds and/or Google Play product SKUs (now called product IDs) that are mapped to this Offering. <br/>- Product managers are to create and add this entry to Offering, if applicable.                                                                                                                                                                                                                                     |
+| Offering         | **Required.** The configuration of the subscription product. This entry includes the Stripe Product ID from Stripe of the product offering, and requires the Capabilities and Purchase entries. If applicable, this entry should be added to a Subgroup if a customer is able to upgrade from or to this product offering (see Subgroup). <br/>- Product managers are to create this entry.                                                                                                  |
```

Comment:
Might be good to mention that if a Product has multiple tiers, each one of those tiers should be a separate product: Basic, Pro, Premium, etc.

Reply:
Thanks Ben! I'll add that into the table. There is a section further down on L75, but it'd be good to have it in both places just in case.